code/modules/awaymissions/corpse.dm

```diff
@@ -425,7 +425,7 @@
 /obj/effect/mob_spawn/human/commander
 	name = "Commander"
 	uniform = /obj/item/clothing/under/rank/centcom_commander
-	suit = /obj/item/clothing/suit/armor
+	suit = /obj/item/clothing/suit/armor/centcom
```

Comment:
This is a full suit of armour like the thunderdome. The commander is supposed to have a vest. Maybe use the bulletproof one.

/obj/item/clothing/suit/armor/bulletproof